packages/nimble-components/src/wafer-map/types.ts

```diff
@@ -0,0 +1,35 @@
+export const WaferColorByOptions = {
```

Comment:
Some naming suggestions:
1. we should be consistent about prefixing these names. This helps clients know which type they're importing, similar to how Nimble has `ButtonAppearance`, `BreadcrumbAppearance`, etc. I'd vote to use `WaferMap` as the prefix to match the name of the component, but if `Wafer` makes more sense semantically then I would be ok with that instead.
2. We don't need "Options" as part of the name since every enum-like type is a set of options

This would lead to these names:
- `WaferMapColorBy`
- `WaferMapQuadrant`
- `WaferMapOrientation`
- `WaferMapDataType`

I'm open to alternate proposals, but I think something like that would be more consistent with other types in Nimble.

Reply:
Ok, sure I will update

Reply:
I updated it